key/store.go

```diff
@@ -143,8 +149,11 @@ func (f *fileStore) SaveShare(share *Share) error {
 	return Save(f.shareFile, share, true)
 }
 
-func (f *fileStore) LoadShare() (*Share, error) {
+func (f *fileStore) LoadShare(sch *crypto.Scheme) (*Share, error) {
 	s := new(Share)
+	if sch != nil {
+		s.Scheme = sch
```

Comment:
if scheme is nil, should we be setting a default here?

Reply:
It's happening in the Share `FromTOML` code if it's `nil`:
https://github.com/drand/drand/blob/65e1329ff3e01b62c78b5290e8936b480e4a58f9/key/keys.go#L325-L334